app/src/features/apiClient/screens/apiClient/components/modals/ImportFromCurlModal/ImportRequestModal.tsx

```diff
@@ -57,9 +70,14 @@ export const ImportRequestModal: React.FC<Props> = ({ isOpen, handleImportReques
       onOk={onImportClicked}
       maskClosable={false}
       confirmLoading={isRequestLoading}
-      onCancel={onClose}
+      onCancel={handleClose}
```

Comment:
I think the `error` also needs to be cleared in other cases as well

it is leading to intermittent behaviors like these 

https://github.com/user-attachments/assets/c79c6b5c-8435-42c4-af06-861166c37556